websites/L/Library Of Babel/metadata.json

```
@@ -0,0 +1,29 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "Library Of Babel",
+	"description": {
+		"en": "A rich presence for libraryofbabel.info"
```

Prefer the official description
```suggestion
		"en": "A project towards a universal library. By this art you may contemplate the variation of the 23 letters."
```